payloads/v9/auditLog.ts

```diff
@@ -650,6 +657,21 @@ export type APIAuditLogChangeKeyDefaultAutoArchiveDuration = AuditLogChangeData<
 	number
 >;
 
+/**
+ * Returned when entity type of a guild scheduled event is changed
+ */
+export type APIAuditLogChangeKeyEntityType = AuditLogChangeData<'entity_type', number>;
+
+/**
+ * Returned when status of a guild scheduled event is changed
+ */
+export type APIAuditLogChangeKeyStatus = AuditLogChangeData<'status', number>;
+
+/**
+ * Returned when location of a guild scheduled event is changed
+ */
```

Comment:
Same as v8

Reply:
I can't find the v8 version of this suggestion, was this a mistake?

Reply:
I probably replied to the wrong line then, basically what I reported in v9 should also be fixed in v8